lib/pwnlib/tubes/tube.rb

```diff
@@ -271,6 +271,22 @@ def recvregex(regex, timeout: nil)
         recvpred(timeout: timeout) { |data| data =~ regex }
       end
 
+      # Receives data until reaching EOF or a timeout is occurred.
+      #
+      # @!macro timeout_definition
+      #
+      # @return [String]
+      #   Returns the data received.
+      def recvall(timeout: nil)
+        recvn(1 << 63, timeout: timeout)
+      rescue ::Pwnlib::Errors::EndOfTubeError, ::Pwnlib::Errors::TimeoutError
+        @buffer.get
+      end
+      alias readall recvall
+      # match Ruby's naming convention
```

Comment:
We don't have similar alias for other methods (recvline, recvpred, ...), is this really needed?

Reply:
hmm, ok